examples/substitutions/BUILD.bazel

```diff
@@ -0,0 +1,36 @@
+load("@doxygen//:doxygen.bzl", "doxygen")
+load("//substitutions:make_var_substitution.bzl", "make_var_substitution")
+load("@bazel_skylib//rules:common_settings.bzl", "string_flag")
+
+genrule(
+    name = "doxyfile_template",
+    srcs = ["Doxyfile.template"],
+    outs = ["NewDoxyfile"],
+    cmd = "sed s/{{PROJECT_NUMBER}}/$$(grep -oP '(?<=BUILD_EMBED_LABEL ).+' bazel-out/stable-status.txt)/ $(SRCS) > $@",
+    cmd_ps = "Get-content $(SRCS) | ForEach-Object { $$_ -replace '{{PROJECT_NUMBER}}', (Select-String -Pattern '(?<=BUILD_EMBED_LABEL +)(.+)' bazel-out/stable-status.txt -AllMatches | Select-Object -Expand Matches | Select-Object -Expand Value) } > $@",
+    stamp = -1,
+)
```

Comment:
Thank you for also considering our usecase. I will use the new release and share feedback how this works out over time.